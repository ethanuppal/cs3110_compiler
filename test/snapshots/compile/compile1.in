func greet(str: Char*) {
    std::print_string("Hello to: ")
    std::print_string(str)
    std::print_endline()
}

func main() {
    print_string("hi\n")
    print_string("hello\n")
    greet("Professor Clarkson")
}
