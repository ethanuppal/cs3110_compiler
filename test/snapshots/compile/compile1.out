hi
hello
Hello to: Professor Clarkson
